doc/dependency_decisions.yml

```diff
@@ -0,0 +1,43 @@
+---
+- - :whitelist
+  - MIT
+  - :who: 
+    :why: 
+    :versions: []
+    :when: 2017-06-23 06:12:33.037106000 Z
+- - :whitelist
+  - Apache 2.0
+  - :who: 
+    :why: 
+    :versions: []
+    :when: 2017-06-23 06:12:54.025599000 Z
+- - :whitelist
+  - Simplified BSD
+  - :who: 
```

Comment:
Let me know if you need these comments or we can generally accept that all BSD/MIT licenses are approved.